pkg/operation/botanist/controlplane.go

```diff
@@ -1169,6 +1171,7 @@ func (b *Botanist) DeployETCD(ctx context.Context) error {
 			}
 
 		case common.EtcdRoleEvents:
+			delete(podAnnotations, "cluster-autoscaler.kubernetes.io/safe-to-evict")
```

Comment:
Not required, Right? `podAnnotation` is newly declared map variable and doesn't include this annotation.

Reply:
It is outside the for loop. So its same for both. Because etcd-main is processed first, the annotation is added. I can rearrange the etcd order; first to reconcile etcd-events and then etcd-main.

Reply:
Got your point. But then don't make it order dependent. Instead please move `podAnnotation` declaration inside for loop.